<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HuboCmaManage</class>
 <widget class="QDialog" name="HuboCmaManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="runCmaBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>runCma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runCmaWithResBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>runCmaWithRes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>load solution</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>save solution</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>31</y>
     <width>114</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>pause cma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resumeBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>91</y>
     <width>114</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>resume cma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>114</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>stop cma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="simulateBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>114</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>simulate</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>iter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>grf ks</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>grf kd</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>PD ks</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>PD kd</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="iterEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="grfKsEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="grfKdEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="PDKdEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="PDKsEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
